### DISCUSSION TITLE
How to stop SumatraPDF from opening 2 windows at once?

### DISCUSSION THREAD

**Discussion by jerouy** [sumatrapdfreader/sumatrapdf#4159] [Q&A] [answered]
Sorry if this has been asked and answered before -- couldn't find any.

So here is my observation:
Let's say I have 3 PDF files: A.pdf; B.pdf; C.pdf
Also I disabled "Use tabs" in Options -- that means each PDF is opened in an individual window.

Steps:
1. Double-click A.pdf -- SumatraPDF opens it in a window.
2. Close this window
3. Double-click B.pdf -- SumatraPDF opens 2 windows at once: A.pdf and B.pdf
4. Close both windows
5. Double-click C.pdf -- SumatraPDF again opens 2 windows at once: A.pdf and C.pdf

I don't want to have A.pdf open with other PDFs -- any settings to control that?

I did find out that if use Ctrl+w to close A.pdf first, then close the remaining SumatraPDF window before double-clicking B.pdf, then only B.pdf is opened.
However, now B.pdf will open every time alongside with other PDFs, until using Ctrl+w to close it first.

I don't believe this behavior happened before current version. v3.4.6 on my other machine doesn't seem to do this.

Thanks,

J

**Comment by GitHubRulesOK** [comment, marked as answer]
it is newer remember session behaviour (previously was 2 or more remembered files on close would restore both as a session)

**Reply by jerouy** [reply]
Cool. I opened SumatraPDF-settings.txt and edited "RestoreSession = false" then all good again.
Thanks for the clue!

**Reply by mailtijmen** [reply]
This solved my problem but I would suggest to automatically set restore session to false if the use tabs option is toggled off. 
Otherwise, selecting 6 pdfs and hitting enter results in those six pdfs all being opened multiple times.
